.eleventy.js

```diff
@@ -30,8 +30,12 @@ module.exports = function (eleventyConfig) {
             return "<pre class=\"highlight\"><code class=\"hljs " + lang + "\">" + markdownit.utils.escapeHtml(str) + "</code></pre>";
         }
     });
+
+    var string = require("string");
```

Comment:
It may make more sense to use the `slugify` package that Eleventy uses for URL generation (see: https://www.11ty.dev/docs/filters/slug/). Note that in Eleventy 1.0, which this site doesn't use yet, they are migrating to `@sindresorhus/slugify`, so maybe adding that dependency and using it will make the Eleventy upgrade cleaner when the time comes).

Reply:
As discussed in the chats, slugify is an ES6 module that doesn't play very nicely with our existing approach.  We decided to use the older slugify built into our version of eleventy.